languages/python/sqlalchemy-oso/tests/test_post_relationship.py

```diff
@@ -644,9 +644,9 @@ def test_in_with_constraints_but_no_matching_objects(
         + " posts.access_level AS posts_access_level,"
         + " posts.created_by_id AS posts_created_by_id, posts.needs_moderation AS posts_needs_moderation"
         + " \nFROM posts"
-        + " \nWHERE (EXISTS (SELECT 1"
+        + " \nWHERE EXISTS (SELECT 1"
```

Comment:
No idea why. SQLalchemy version thingy?

Reply:
Was this failing without your change? Or after the change?

Reply:
I can check. I suspect there's an additional `and` that was stripped away from the query filter. e.g. `and(and(...))` vs `and(...)`